src/com/javaninjas/blackjack/service/Player.java

```diff
@@ -22,11 +22,28 @@
     private boolean hasBlackJack = false;
 
     //Constructors
+
```

Comment:
This constructor doesn't retrieve the welcome ascii art

Reply:
made changes